analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -226,7 +228,7 @@ abstract class PackageManager(
         definitionFiles.forEach { definitionFile ->
             log.info { "Resolving $managerName dependencies for '$definitionFile'..." }
 
-            val duration = measureTime {
+            val duration = measureTimeMillis {
```

Comment:
We should then also drop the `, "-Xopt-in=kotlin.time.ExperimentalTime"` part from https://github.com/oss-review-toolkit/ort/blob/f587d6315a595778edfbda3ccb8eee0e9d1d8bbf/build.gradle.kts#L186

Reply:
Done.